code/ConvertToStandardPath_SubmarineCable.py

```diff
@@ -217,35 +216,32 @@ def get_data_from_database(db_file):
     """
     cursor.execute(sql_query)
     datas = cursor.fetchall()
-    wktToCities = {}
-    cableIdToWkt = {}
+    cable_id_to_cities = {}
+    cable_id_to_wkt = {}
     for cable_id, cable_wkt, _, _, _, _, _ in datas:
-        if cable_id in cableIdToWkt:
-            for path in convert_multilinestring_to_list(cable_wkt):
-                cableIdToWkt[cable_id].add(str(path))
-        else:
-            cableIdToWkt[cable_id] = set()
-            for path in convert_multilinestring_to_list(cable_wkt):
-                cableIdToWkt[cable_id].add(str(path))
+        if cable_id not in cable_id_to_wkt:
+            cable_id_to_wkt[cable_id] = set()
+        for path in convert_multilinestring_to_list(cable_wkt):
+            cable_id_to_wkt[cable_id].add(str(path))
 
     for cable_id, _, city_name, city_state, city_country, city_latitude, city_longitude in datas:
-        if cable_id in wktToCities:
-            wktToCities[cable_id].add(
+        if cable_id in cable_id_to_cities:
+            cable_id_to_cities[cable_id].add(
                 (city_name, city_state, city_country, city_latitude, city_longitude))
         else:
-            wktToCities[cable_id] = set()
-            wktToCities[cable_id].add(
+            cable_id_to_cities[cable_id] = set()
```

Comment:
Can you also update this like the above if else statement.

Reply:
fixed, I also check other if-else statement in this code, and also fixed the same issue for function verify_graph_with_cities.